crates/node-bindings/src/parcel/parcel.rs

```diff
@@ -41,6 +45,19 @@ impl ParcelNapi {
     })
   }
 
+  #[napi]
+  pub async fn build(&self) -> napi::Result<()> {
+    self.parcel.build().unwrap();
+    Ok(())
+  }
+
+  #[napi]
+  pub fn default_thread_count(env: Env) -> napi::Result<JsNumber> {
+    let cpus = num_cpus::get();
+    let cpus = env.create_int32(cpus as i32)?;
+    Ok(cpus)
+  }
```

Comment:
The nodejs worker count and rust thread count are decoupled from each other.

Parcel will default to spawning the same number of nodejs workers threads as the specified Rust threads. Users can optionally configure more/less Nodejs workers.

```javascript
new Parcel({
  threads: undefined, // defaults to available processors
  nodeWorkers: // defaults to the value of "threads",
})
```